twisted/web/topfiles/8685.bugfix

```diff
@@ -0,0 +1 @@
+twisted.web._http2.H2Connection now swallows header writes on reset streams, rather than erroring out.
```

Comment:
hm... we are referencing a private API. Can this be redone so that we only talk about public API in the release notes?